book/src/custom-extensions/algebra.md

```diff
@@ -26,11 +29,15 @@ To [leverage](./overview.md) compile-time known moduli for performance, you decl
 ```rust
 moduli_declare! {
     Bls12_381Fp { modulus = "0x1a0111ea397fe69a4b1ba7b6434bacd764774b84f38512bf6730d2a0f6b0f6241eabfffeb153ffffb9feffffffffaaab" },
-    Bn254Fp { modulus = "21888242871839275222246405745257275088696311157297823662689037894645226208583" },
+    Bn254Fp {
+        modulus = "21888242871839275222246405745257275088696311157297823662689037894645226208583", impl_field = true
```

Comment:
I am not quite a fan of the `impl_field` field, though I could live with it if we needed it. But I think it might actually be easy inside the proc-macro to just check if the modulus as a big-int is prime, so `impl_field` could be inferred?